### ISSUE TITLE
Add rule to disallow inline styles

### ISSUE THREAD

**Issue by marekdedic** [sveltejs/eslint-plugin-svelte#603]
### Motivation

When using CSP, you often want to disallow inline styles (to not have `style-src: 'unsafe-inline'`), I'd like to add a rule that checks this automatically

### Description

The rule would look for any directive producing inline styles and error on them.

Additionaly, using svelte transitions generates inline styles (although that may change in the future if/when svelte migrates to teh web animations - see sveltejs/svelte#6662), so this rule should probably ban using them altogether (gated with a configuration option probably?)

### Examples

```svelte
<!-- ✗ BAD -->
<div style="anything" />
<div style:anything={true} />
<div transition:anything />
```

**Comment by ota-meshi** [comment]
Thank you for suggesting new rules! That rule sounds good to me!
